src/data/full-quiz.js

```diff
@@ -1,4 +1,13 @@
 const fullQuiz = [
+  {
+    Question: "Select the right syntax of the Arrow function (ES6)?",
+    Answer: "let variable = () => {//code block}",
+    Distractor1: "let variable => (){//code block}",
+    Distractor2: "let variable = function() => {//code block}",
+    Distractor3: "let variable = function(){//code block}",
+    Explanation: "ES6 arrow functions provide you with an alternative way to write a shorter syntax compared to the function expression.",

```

Comment:
proposed fix:
ES6 arrow functions provide you with an alternative way to write shorter syntax compared to the traditional function expression.